Mage.Sets/src/mage/sets/CrimsonVowCommander.java

```diff
@@ -136,6 +136,8 @@ private CrimsonVowCommander() {
         cards.add(new SetCardInfo("Spectral Sailor", 114, Rarity.UNCOMMON, mage.cards.s.SpectralSailor.class));
         cards.add(new SetCardInfo("Spectral Shepherd", 98, Rarity.UNCOMMON, mage.cards.s.SpectralShepherd.class));
         cards.add(new SetCardInfo("Stensia Masquerade", 150, Rarity.UNCOMMON, mage.cards.s.StensiaMasquerade.class));
+        cards.add(new SetCardInfo("Storm of Souls", 9, Rarity.RARE, mage.cards.s.StormOfSouls.class, NON_FULL_USE_VARIOUS));
+        cards.add(new SetCardInfo("Storm of Souls", 47, Rarity.RARE, mage.cards.s.StormOfSouls.class, NON_FULL_USE_VARIOUS));
```

Comment:
Sorry, one last thing.  What is NON_FULL_USE_VARIOUS?  I don't see tags like that on any other card for this set.

Reply:
To be honest, i don't know. 

I've seen it on a few other cards. And iirc, it complains when I try to add the same card twice without. I am not sure how to properly all versions of a card to a set.

Reply:
See for example all of AFR, https://github.com/magefree/mage/blob/master/Mage.Sets/src/mage/sets/AdventuresInTheForgottenRealms.java

Reply:
Ah ok.  Yea, I usually don't bother with alt-art versions of cards.  That gets taken care of when the set is complete.  It looks like this is probably fine though.